tests/test_ingesters.py

```diff
@@ -695,6 +694,58 @@ def test_parameter_assignment_for_attributes(self):
         self.assertEqual(created_dataset_uri, False)
 
 
+class FTPIngesterTestCase(django.test.TestCase):
+    """Test the FTPIngester"""
+
+    def setUp(self):
+        self.patcher_param_count = mock.patch.object(Parameter.objects, 'count')
+        self.mock_param_count = self.patcher_param_count.start()
```

Comment:
This patcher must be stopped, either in `tearDown()` or in a cleanup.

Reply:
Done in the next commit